modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -232,18 +253,58 @@ For more information, see
 ====
 
 
+ifeval::[{suma-content} == true]
+== Install {productname} on {sles} {bci-suma}
+
+Alternatively, you can install {productname} on {sles} {bci-suma}.
+The following procedure describes the main steps of the installation process.
+
+.Procedure: Installing {productname} on {sles} {bci-suma}
+. Locate and download {sles} {bci-suma} [literal]``.iso`` at https://www.suse.com/download/sles/.
+. Make sure that the {scc} code you plan to use has the Extensions enabled.
+. Start the installation of {sles} {bci-suma}.
+  .. On the [literal]``Language, keyboard and product selection`` select the product to install.
+  .. On the [literal]``License agreement`` read the agreement and check [guimenu]``I Agree to the License Terms``.
+. Select the registration method. 
+. In the screen [literal]``Extensions and Modules Selection`` check the following:
+  * {productname} Server Extension
+  * Basesystem Module
+  * Containers Module
+. Complete the installation.
+. When the installation completes, log in to the newly installed server as root.
+. As root, run
+
++
+
+[source,shell]
+----
+zypper install podman
+----
```

Comment:
@0rnela I'm not completely sure if those steps are complete. 

podman and the other dependencies are in the container module.  But it will be good  to check if podman is installed  or no